src/pages/demo/personProfiles/index.tsx

```diff
@@ -0,0 +1,56 @@
+import { drupalClient } from '@/utils/drupalClient'
+import { PersonProfile } from '@/components/node/person_profile'
+import { DrupalJsonApiParams } from 'drupal-jsonapi-params'
+import Container from '@/components/container'
+
+const PersonProfilePage = ({ nodes }) => {
+  if (!nodes) return
+  return (
+    <>
+      <Container className="container">
+        <div id="content" className="interior">
+          <main className="va-l-detail-page va-facility-page">
+            <div className="usa-grid usa-grid-full">
+              <div className="usa-width-three-fourths">
+                {nodes.map((node, index) => (
+                  <div key={index}>
+                    <PersonProfile node={node} />
+                  </div>
+                ))}
+              </div>
+            </div>
+          </main>
+        </div>
+      </Container>
+    </>
+  )
+}
+
+export default PersonProfilePage
+
+export async function getStaticProps(context) {
+  const params = new DrupalJsonApiParams()
+  params
+    .addFilter('status', '1')
+    .addFilter('field_office', null, 'IS NOT NULL')
+    .addFilter('field_media', null, 'IS NOT NULL')
+    .addInclude(['field_office', 'field_media'])
+
+  const nodes = await drupalClient.getResourceCollectionFromContext(
+    'node--person_profile',
+    context,
+    {
+      params: {
```

Comment:
Note that this is the same issue I found I was doing incorrectly earlier: this creates the `params` variable above but then doesn't use it. If we want to use the `DrupalJsonApiParams()` object (and I think it might be easier to work with than constructing the params directly as JSON), you could do the things here - adding the thumbnail, the media image, etc - up above, and then replace this with:

```
params.getQueryObject()
```

which will generate an object like this.